### ISSUE TITLE
Datatypes

### ISSUE THREAD

**Issue by andymans** [brianc/node-postgres#1297]
Mate, you need a 'datatype' section on your Wiki. Everyone who's involved in RDBMS space has one...and there's a good reason for it.

I can understand I might need to spend an hour googling to learn how to do some esoteric piece of SQL weirdness - but not...how to insert a TIMESTAMPZ.

**Comment by brianc** [comment]
Good call! I'm overhauling the documentation __majorly__ in the next few weeks. I'll be sure to add a note to add this as well!

**Comment by brianc** [comment]
if you don't mind I'd like to leave this issue open so I can make sure it's documented in the new documentation better. I've added it to the docs milestone & it incorrectly reports as progress on the milestone though I haven't finished it yet.

**Comment by charmander** [comment]
https://node-postgres.com/features/types/ covers the parsing half of this so far. Related: #1462
